Chore: Update readme to expand new options and query selection

academo opened the pull request:
Updates the outdated readme that didn't contain all the available options and updated option names.

Also adds a queries section and troubleshooting for problems like https://github.com/grafana/clock-panel/issues/283 where the query throws error and the user doesn't know how to fix them.

academo committed:
Chore: Update readme to expand new options and query selection

sympatheticmoose reviewed (commented):
Generally the content is good, but not sure about the flow and feel like things are jumbled. Generally I think its helpful when we mirror the UI for ordering

academo committed:
Update readme

academo commented on README.md:
```diff
@@ -11,70 +11,125 @@ Show the time in another office or show a countdown/countup to an important even
+- The selected datasource has an empty or invalid query that produces an error
 
-To build a production build with minification: `yarn build`
+**Solution**: If you're not using queries, select a datasource that won't generate errors:
```
that's also a good option

urbiz-grafana reviewed (changes requested):
Can you capitalize Make and Install (steps 2 and 3)? @academo

academo committed:
captilize steps

urbiz-grafana committed:
Update README.md

TW review

@academo FYI - just suggestions, feel free to turn them down. LMK if you have any questions!
